plsync/planetlab/model_test.py

```diff
@@ -95,6 +142,31 @@ def test_slice_dnsname(self):
         self.assertEqual('def.abc', twoname)
         self.assertEqual('def.ghi.abc', multiname)
 
+    def test_slice_sitename(self):
+        names = []
+        expected_name = 'bar.abc.abc01'
+
+        for experiment in self.experiments:
+            for _, node in experiment['network_list']:
+                names.append(experiment.sitename(node))
+
+        self.assertEqual(1, len(set(names)))
```

Comment:
The sitename for an experiment will be the same for all nodes at a site. "set" collapses duplicates, and there should only be one unique name left.

For example, ndt.iupui.nuq02.measurement-lab.org resolves to all IPs assigned to the experiment at that site.

I've added a comment to preserve the thinking here.

Look good?